### DISCUSSION TITLE
VCA

### DISCUSSION THREAD

**Discussion by G25MOM** [Mogli12/KeyboardSteer#863] [Ideas]
Moin mogli. beim vca gibt es ja die wahl ds man seine geschwindigkeit halten kann.. aber man dadurch kein gas mehr geben kann.. wäre das ganze möglich so umzuschreiben das man das auch wieder kann.. und z.b man hat eine geschwindigkeit von 15km/h vorgegeben unnd gibt man z.b kein gas geht er runter auf 5 km/H und mit vollgas z.b könnte er bis 20 km/h fahren ? weil das würde so wirklich sehr nahe am fendt vario ran kommen.. und damit hätte man nun wirklich mal ein vario getriebe im ls

**Comment by Mogli12** [comment]
Ich habe in Build 118 einen Geschwindigkeitsbegrenzer eingebaut. Wenn er aktiv ist, dann wird das über dem Tacho mit dem Text "speed limiter" angezeigt. Die maximale Geschwindigkeit ist die vom Tempomaten. Das ist zwar nicht genau das, was du dir gewünscht hast, aber hoffentlich nah genug dran. Die stufenlosen Getriebe laufen ja recht langsam aus.
Das schaltet man mit der Kombination KEY_lctrl KEY_semicolon (Links STRG + Ö auf deutscher Tastatur) an und aus.

**Reply by G25MOM** [reply]
und da war ja die frage ob man das so umbauen oder umschreiben kann wie ich oben beschriebe habe

**Comment by Mogli12** [comment]
Na ja, die Einstellung mit der maximalen Geschwindigkeit war einfach einzubauen. Alles andere ist mir persönlich zu kompliziert, vor allem da ich nicht so recht verstehe wie Fendt das macht.
Anstatt von 5 bis 20 km/h bei der Einstellung 15 km/h kannst Du so von 0 bis 20 km/h bei der Einstellung 20 km/h beschleunigen. Der Traktor regelt dabei die Drehzahl selber. Das ist doch TMS, oder?

**Comment by G25MOM** [comment]
Nee tms ist bisschen anders noch

Sonst falls du lust hast können wir uns gerne mal aufn DC unterhalten.
Konnte ich dir das vario getriebe bisschen besser erklären und ggf auch
gerne Videos machen


Stefan Biedenstein ***@***.***> schrieb am Di., 25. Okt.
2022, 20:32:

**Comment by Mogli12** [comment]
Hallo,
ich habe den Geschwindigkeitsbegrenzer etwas erweitert. Nun kann man mit einem analogen Gaspedal die Geschwindigkeit mehr oder weniger direkt vorwählen.
Wenn der Tempomat auf 10 km/h steht, ausgeschaltet ist, und der Begrenzer an ist, dann ergibt 50% Gaspedal 5 km/h. Bei 0% Gaspedal rollt der Schlepper langsam bis zum Stillstand aus (ca. 2,5 km/h pro Sekunde weniger) 
Kannst Du mal Build 122 testen? Es ist zwar immer noch nicht wie von Dir beschrieben, aber vielleicht ist es ja besser?
Viele Grüße

**Comment by G25MOM** [comment]
Moin Freund mich wirklich sehr das du dir die Aufgabe angenommen hast :)
werde das gleich mal testen:)

Stefan Biedenstein ***@***.***> schrieb am Mi., 11. Jan.
2023, 17:41:

**Comment by G25MOM** [comment]
Also ich habe das nun oder bin grade dabei zu testen. Ansich ist das schon
der richtige Weg nur was nicht ganz stimmt ist halt das man z.b 20km
vorgibt er aber auch nicht schneller als 20 km/h fährt was er ja eigentlich
kann sobald ich Gas gebe

Nico ***@***.***> schrieb am Mi., 11. Jan. 2023, 18:03:

**Comment by G25MOM** [comment]
Das müsste man eigentlich beim geschwindigkeit permanent halt können z.b
steht da 25 halten kann aber mit voll Gas bis zu 30km/h fahren und ohne Gas
20 km/h

Man müsste sich mal unterhalten können ist das ganze bisschen leichter
zuerklären

Nico ***@***.***> schrieb am Mi., 11. Jan. 2023, 18:28:
